tests/unittests/Foundation/NSNotificationTests.mm

```diff
@@ -15,7 +15,54 @@
 //******************************************************************************
 
 #include <TestFramework.h>
+#import <Starboard/SmartTypes.h>
 #import <Foundation/Foundation.h>
+
+// Avoid "property access result unused" warnings by _consume-ing the value of a property accessor.
+template <typename... T>
+static inline void _consume(T&&...) {
+}
+
+TEST(NSNotification, CanCreate) {
+    EXPECT_OBJCNE(nil, [NSNotification notificationWithName:@"name" object:nil userInfo:nil]);
+}
+
+@interface BadNSNotificationSubclass : NSNotification

```

Comment:
I will cap this review with a reformat, but I don't want to churn unchanged lines.